migrations/1727186561690_idx-contract-call-function-name-trgm.js

```diff
@@ -0,0 +1,19 @@
+/** @param { import("node-pg-migrate").MigrationBuilder } pgm */
+exports.up = pgm => {
+  pgm.sql('CREATE EXTENSION IF NOT EXISTS pg_trgm;');
```

Comment:
API deployers might not have this extension installed. You should validate first with 
```sql
SELECT * FROM pg_available_extensions WHERE name = 'pg_trgm';
```
and also consider this possibility in your query functions